packages/cli/templates/webcomponents/igc-ts/projects/side-nav/index.ts

```diff
@@ -0,0 +1,18 @@
+import { ProjectTemplate } from "@igniteui/cli-core";
+import * as path from "path";
+import { BaseIgcProject } from "../_base";
+
+export class SideNavProject extends BaseIgcProject implements ProjectTemplate {
+	public id: string = "webcomponent";
```

Comment:
id "webcomponent"? That's affects the CLI parameters of `ig new`, pretty sure it should be `side-nav` for example. Also should be reflected in the docs/wiki

Reply:
I changed it and updated the wiki